test/models/refund_test.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RefundTest < ActiveSupport::TestCase
+  def setup
+    @refund = refunds(:one)
+  end
+
+  test 'destroy refund should destroy articles_refunds' do
+    assert_difference 'Refund.count', -1 do
+      @refund.destroy
+    end
+  end
+
+  test 'destroy refun should destroy refunds_subscription_offers' do
+    assert_difference 'Refund.count', -1 do
+      @refund.destroy
+    end
+  end
```

Comment:
I don't think the tests are doing what their names are implying. We are testing that `@refund.destroy` deletes the refund, not the associated records